gradlew.bat

```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off
```

Comment:
Could this file be committed again in this PR to show only the changes so that the changes can be reviewed, as of now it is showing as if the entire file has changed?